apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryHandler.cs

```diff
@@ -36,11 +37,8 @@ internal sealed class RetryHandler : IHttpUnsuccessfulResponseHandler
         internal static void MarkAsRetriable<TResponse>(StorageBaseServiceRequest<TResponse> request) =>
             request.AddUnsuccessfulResponseHandler(s_instance);
 
-        //This function is designed to be asynchronous in case we need to examine the response content, but for now we only need the status code
-        internal static Task<bool> IsRetriableResponse(HttpResponseMessage response)
-        {
-            return Task.FromResult(s_retriableErrorCodes.Contains(((int)response.StatusCode)));
-        }
+        // This function is designed to support asynchrony in case we need to examine the response content, but for now we only need the status code
+        internal static Task<bool> IsRetriableResponse(HttpResponseMessage response) => Task.FromResult(s_retriableErrorCodes.Contains(((int)response.StatusCode)));
```

Comment:
This is fine - but I'd have put the actual implementation on a new line to avoid the line being quite so long. Apologies if I wasn't clear in my previous comment. We don't have hard-and-fast line length rules, but this is currently 165 characters which is longer than I'd normally like. (I'm sure you can find counterexamples though!)